package.json

```diff
@@ -157,28 +158,28 @@
     "stylelint": "^13.13.1",
     "stylelint-config-recommended-scss": "^4.3.0",
     "stylelint-scss": "^3.21.0",
+    "ts-patch": "^3.1.2",
+    "typescript": "^5.4.5",
     "webpack-bundle-analyzer": "^4.5.0",
     "whatwg-fetch": "^3.6.2"
   },
   "scripts": {
-    "build": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
+    "build": "fec build",
     "test": "TZ=UTC jest --verbose --no-cache",
     "test:local": "TZ=UTC jest --verbose --collectCoverage=false",
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint config src",
     "lint:js:fix": "eslint config src --fix",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelint.json",
-    "prod": "NODE_ENV=production webpack serve --config config/dev.webpack.config.js",
     "server:ctr": "node src/server/generateServerKey.js",
-    "start": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
-    "start:proxy": "PROXY=true NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
-    "start:proxy:beta": "BETA=true PROXY=true NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
-    "travis:build": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
+    "start": "fec dev",
+    "start:proxy": "PROXY=true fec dev",
+    "travis:build": "NODE_ENV=production webpack --config fec.webpack.config.js",
```

Comment:
We can remove this task too. and just replace `travis:build` with just `build` in `travis:verify`

Reply:
are you saying to replace with `travis:verify : travis build` or fec:build?

Reply:
> replace `travis:build` with just `build` in `travis:verify`

Below the `travis:verify` calls `travis:build`. This can be replaced with just `build` and the `travis:build` task can be removed.